source/Application-Starter-Migration-V1-V2/ApplicationStarterCommandLineHandler.extension.st

```diff
@@ -0,0 +1,145 @@
+Extension { #name : #ApplicationStarterCommandLineHandler }
+
+{ #category : #'*Application-Starter-Migration-V1-V2' }
+ApplicationStarterCommandLineHandler >> currentTimeStampString [
+
+	self
+		deprecated: 'Please use the class side method instead'
+		on: '2019-11-15'
+		in: 'v2'
+		transformWith: '`@receiver currentTimeStampString' -> '`@receiver class currentTimeStampString'.
+	^ self class currentTimeStampString
+]
+
+{ #category : #'*Application-Starter-Migration-V1-V2' }
+ApplicationStarterCommandLineHandler >> defaultLogFile [
+
+	self
+		deprecated: 'Please use the class side method instead'
+		on: '2019-11-15'
+		in: 'v2'
+		transformWith: '`@receiver defaultLogFile' -> '`@receiver class defaultLogFile'.
+	^ self class defaultLogFile
+]
+
+{ #category : #'*Application-Starter-Migration-V1-V2' }
+ApplicationStarterCommandLineHandler >> defaultLogsPath [
+
+	self
+		deprecated: 'Please use the class side method instead'
+		on: '2019-11-15'
+		in: 'v2'
+		transformWith: '`@receiver defaultLogsPath' -> '`@receiver class defaultLogsPath'.
+	^ self class defaultLogsPath
+]
+
+{ #category : #'*Application-Starter-Migration-V1-V2' }
+ApplicationStarterCommandLineHandler >> defaultStackDumpFile [
+
+	self
+		deprecated: 'Please use the class side method instead'
+		on: '2019-11-15'
+		in: 'v2'
+		transformWith: '`@receiver defaultStackDumpFile' -> '`@receiver class defaultStackDumpFile'.
+	^ self class defaultStackDumpFile
+]
+
+{ #category : #'*Application-Starter-Migration-V1-V2' }
+ApplicationStarterCommandLineHandler >> dumpStackAndReport: aSignal [
+
+	self
+		deprecated: 'Please use the class side method instead'
+		on: '2019-11-15'
+		in: 'v2'
+		transformWith: '`@receiver dumpStackAndReport: `@signal' -> '`@receiver class dumpStackAndReport: `@signal'.
+	^ self class dumpStackAndReport: aSignal
+]
+
+{ #category : #'*Application-Starter-Migration-V1-V2' }
+ApplicationStarterCommandLineHandler >> logError: anErrorMessage [
+
+	self
+		deprecated: 'Please use CurrentLogger value error: instead'
+		on: '2019-11-15'
+		in: 'v2'
+		transformWith: '`@receiver logError: ̀@message' -> '`CurrentLogger value error: `@message'.
+
+	CurrentLogger value error: anErrorMessage
```

Comment:
This should have been renamed as `logAsError:` (same for all other message sends to CurrentLogger in this extension)

Reply:
:+1 I was missing the unloaded packages.